src/examples/checkbox/CheckboxExamples.tsx

```diff
@@ -0,0 +1,65 @@
+import { GoACheckbox, GoAFormItem, GoAInput } from "@abgov/react-components";
+import { Sandbox } from "@components/sandbox";
+import { CodeSnippet } from "@components/code-snippet/CodeSnippet.tsx";
+
+export default function CheckboxExamples () {
+    
+    return (
+        <>
+        <h2 id="component-examples" className="hidden" aria-hidden="true">Examples</h2>
+        <h3 id="component-example-expand-collapse-form">Use a component as a description</h3>  
+        <Sandbox fullWidth skipRender>
+            {/*Angular*/}
+            <CodeSnippet
+            lang="typescript"
+            tags="angular"
+            allowCopy={true}
+            code={`
+                <goa-form-item label="How would you like to be contacted?">
+                    <goa-checkbox checked="true" name="item" text="Phone">
+                        <div slot="description">
+                            <goa-form-item mt="m" label="Phone number">
```

Comment:
I noticed that the Input docs have change events. Should I include them here too?